File: DotnetApi/Services/IQuotesService.cs
using DotnetApi.Entities;

namespace DotnetApi.Services;

public interface IQuotesService
{
    public Task<IEnumerable<Quote>> GetQuotes(int length);
    public Task<IEnumerable<Quote>> GetAllQuotes();
    public Task<Quote?> GetRandomQuote();
}

File: KotlinApi/src/main/kotlin/no/dunder/mifflin/dundermifflinquotes/service/IQuotesService.kt
package no.dunder.mifflin.dundermifflinquotes.service

import no.dunder.mifflin.dundermifflinquotes.domain.dto.Quote

interface IQuotesService {

    fun getQuotes(size: Int): List<Quote>

    fun getAllQuotes(): List<Quote>

    fun getRandomQuote(): Quote?
}